publish/scripts/installer.js

```diff
@@ -463,13 +463,18 @@ repositories {
 def supportVersion = project.hasProperty("supportVersion") ? project.supportVersion : "26.0.0"
 def googlePlayServicesVersion = project.hasProperty('googlePlayServicesVersion') ? project.googlePlayServicesVersion : "12.0.1"
 
+if ( VersionNumber.parse( googlePlayServicesVersion ) < VersionNumber.parse( '9.2' ) ) {
+    throw new GradleException('Google Play services version too low, please update to at least 9.2');
```

Comment:
❤️

Reply:
I think I'll add also add a warning in case the version is lower than `googlePlayServicesVersion`